markdown/developers/pipeline-release.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Pipeline Release Guidelines
+subtitle: Guidelines on releasing a sanger-tol pipeline
+---
+
+## nf-core Release Checklist
+
+This page is heavily inspired by the [nf-core release checklist](https://nf-co.re/docs/contributing/release_checklist).
+
+## Branch model
+
+First a reminder about how we use branches in sanger-tol:
+
+- `main` is the release branch. Releases can only happen from the `main` branch and code present on the `main` branch _has_ to be released.
+- `dev` is the staging branch. Code on the `dev` branch should always pass tests and be functional (this is a condition of our [review guidelines](/docs/contributing/review_checklist)). Releasing a pipeline involves merging `dev` into `main`.
+- Feature branches are where development happens.
+
+## Release steps
+
+Follow the "Before you release" and "Steps to release" from the [nf-core release checklist](https://nf-co.re/docs/contributing/release_checklist) with the following adaptations:
+
+1. Replace "nf-core" with "sanger-tol".
+2. Since we develop directly off the sanger-tol repository, the version bump happens there, not "on your fork".
+3. The "core team member" who can activate the integration with Zenodo is @muffato.
```

Comment:
```suggestion
3. The "core team member" who can activate the integration with Zenodo is [@muffato](https://github.com/muffato).
```

Reply:
Add @DLBPointon here as well?

Reply:
For now, I want it to be just me in the docs.
cf TOLIT-1871 the problem we have is that the GitHub integrations are all owned by different people. In that ticket I'll try to find a solution (I hope something along the lines of having a tolsoft account on Zenodo and asking the Zenodo helpdesk to move all integrations over to tolsoft)